Patch readme (#311)
GoekeLab/bambu: README.md

|verbose|A logical if more information should be printed whilst the function is running|

### Quantification of gene expression

> To obtain gene expression, simply summing up over all annotated transcripts will likely underestimate it, as Bambu assign reads to transcripts if they are compatible to the transcripts.

Reviewer: ```suggestion
To obtain gene expression, simply summing up over all annotated transcripts will likely underestimate it, as Bambu assigns only reads to transcripts if they are compatible. Reads which are incompatible with transcripts, but which can be assigned to the gene are tracked by Bambu to obtain more accurate gene expression estimate.
```

> You will need to run the following step to obtain the accurate gene expression, which uses all reads that can be assigned to the transcripts of each gene, including reads that are incompatible with all existing annotations.

Reviewer: ```suggestion
To obtain the accurate gene expression estimates which uses all reads that can be assigned to each gene (including reads that are incompatible with all existing annotations) you can run the following command:
```

```rscript

> SeGene <- transcriptToGeneExpression(se)

Reviewer: ```suggestion
seGene <- transcriptToGeneExpression(se)
```
